static/js/marker.js

```diff
@@ -127,8 +127,9 @@ var MarkerView = Backbone.View.extend({
 
     },
     unhighlight : function() {
-    	if (app.oms.markersNearMarker(this.marker, true)[0]){
-    		this.marker.setIcon(this.getIcon("multiple", SEVERITY_VARIOUS));
+    	if (app.oms.markersNearMarker(this.marker, true)[0] && !this.model.get("currentlySpiderfied")){
+            var group = this.model.get("groupID") -1;
```

Comment:
I don't have any objections to raise.
I believe it is a matter of what feels more intuitive, which to me is
obviously to avoid having a group 0. But I don't see any problem with
changing it, code wise.
Please do feel free to change it accordingly.

2015-04-11 9:19 GMT+03:00 Daniel Hershcovich <notifications@github.com>:

> In static/js/marker.js
> <https://github.com/hasadna/anyway/pull/205#discussion_r28192797>:
>
> > @@ -127,8 +127,9 @@ var MarkerView = Backbone.View.extend({
> >
> >      },
> >      unhighlight : function() {
> > -    	if (app.oms.markersNearMarker(this.marker, true)[0]){
> > -    		this.marker.setIcon(this.getIcon("multiple", SEVERITY_VARIOUS));
> > +    	if (app.oms.markersNearMarker(this.marker, true)[0] && !this.model.get("currentlySpiderfied")){
> > +            var group = this.model.get("groupID") -1;
>
> Why subtract 1? Maybe group IDs should just start from 0 and then they're
> exactly the index in the array.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hasadna/anyway/pull/205/files#r28192797>.
>